.ci/static-checks.sh

```diff
@@ -537,8 +537,25 @@ check_docs()
 		fi
 
 		# Check the URL, saving it if invalid
-		( curl -sLf -o /dev/null "$url" ||\
-				echo "$url" >> "$invalid_urls") &
+		(
+			local http_statuses
+
+			# A single doc can generate multiple HTTP status
+			# codes, so save them all.
+			http_statuses=$(curl -sIL "$url" | grep -E "^HTTP" | awk '{print $2}' || true)
```

Comment:
You dont need the trailing `|| true` here as that's being executed in a subshell.

Reply:
You are right, still I always try to avoid the `|| true` because they look so odd :laughing:

Reply:
And, in this case, if the pipeline fails, why are you ignoring the failure?

Reply:
@marcov - I've broken up that line into multiple sections for clarity and added comments.